bin/probot-run.js

```diff
@@ -26,7 +26,7 @@ if (!program.privateKey) {
   program.privateKey = findPrivateKey()
 }
 
-if (program.tunnel) {
+if (program.tunnel && !process.env.DISABLE_TUNNEL) {
```

Comment:
LGTM ✨ 

Asking questions for the sake of it, what do you think about changing it to the following? Makes it a little more predicable. Only `DISABLE_TUNNEL=true` will _truly_ disable it.

```
if (... && process.env.DISABLE_TUNNEL !== true)
```

Reply:
I think that environment variable would actually be a string.

Reply:
Yeah, as @wilhelmklopp points out, regardless to what the environment variable is set to, it will be a string, so setting it to anything should evaluate to true.